wallet/update.go

```diff
@@ -201,35 +201,32 @@ func appliedEvents(cau chain.ApplyUpdate, walletAddress types.Address) (events [
 					continue
 				}
 
-				outputID := types.FileContractID(fce.ID).MissedOutputID(i)
+				outputID := fce.ID.MissedOutputID(i)
 				sce, ok := siacoinElements[outputID]
 				if !ok {
 					panic("missing siacoin element")
 				}
 
-				addEvent(types.Hash256(types.FileContractID(fce.ID).MissedOutputID(0)), EventTypeV1ContractResolution, EventV1ContractResolution{
+				addEvent(types.Hash256(fce.ID.MissedOutputID(0)), EventTypeV1ContractResolution, EventV1ContractResolution{
```

Comment:
Huh. This should be `MissedOutputID(i)` or just `outputID`.  Wanted to note it, don't mind resolving in a follow up.